cloudfront/wellcomelibrary.org/edge-lambda/src/redirect.ts

```diff
@@ -0,0 +1,14 @@
+import {CloudFrontResultResponse} from "aws-lambda/common/cloudfront";
+
+export async function redirect(uri: string){
+    return {
+        status: '301',
```

Comment:
Two minor thoughts:

* I wonder if we should stick to 302 until we've got the redirection stuff sorted, so clients don't end up caching the redirects?
* Why is this function async?

Reply:
- Good idea on 302.
- The function need not be async.